imgui-SFML.cpp

```diff
@@ -785,6 +806,25 @@ bool ImageButton(const sf::Sprite& sprite, const sf::Vector2f& size, const int f
                               toImColor(bgColor), toImColor(tintColor));
```

Comment:
For example, I'm not sure how this compiles with ImGui 1.89+

Reply:
This should compile fine [considering this](https://github.com/ocornut/imgui/blob/v1.89/imgui_widgets.cpp#L1083)